packages/contracts/README.md

```diff
@@ -0,0 +1,51 @@
+Contracts
+-----
+
+## Useful 0x Wiki Articles
+
+* [Architecture](https://0xproject.com/wiki#Architecture)
+* [Contract Interactions](https://0xproject.com/wiki#Contract-Interactions)
+* [Contract deployed addresses](https://0xproject.com/wiki#Deployed-Addresses)
+* [0x Protocol Message Format](https://0xproject.com/wiki#Message-Format)
+* [Bug Bounty Program](https://0xproject.com/wiki#Bug-Bounty)
+
+## Setup
+
+### Installing Dependencies
+
+Install [Node v6.9.1](https://nodejs.org/en/download/releases/)
```

Comment:
Either remove the version or link to https://nodejs.org/download/release/v6.9.1/